apps/frontend/src/components/App.tsx

```diff
@@ -43,10 +55,17 @@ export default function App() {
           >
             <MenuIcon />
           </IconButton>
-          <Typography variant="h6" component="div" sx={{ flexGrow: 1 }}>
+          <Typography
+            variant="h6"
+            component="div"
+            sx={{ flexGrow: 1 }}
+            onClick={routeHome}
+          >
             Habit Helper
           </Typography>
-          <Button color="inherit">Login</Button>
+          <Button color="inherit" onClick={routeLogin}>
```

Comment:
This can be replaced with:

```jsx
<Button color="inherit" component={RouterLink} to="/login">
```

In this case RouterLink would be an alias to the React Router Link component ie. 
```jsx
import {
  Route,
  Routes,
  Link as RouterLink,
} from "react-router-dom";
```

This should simplify code a little.